pty.go

```diff
@@ -0,0 +1,57 @@
+package ssh
+
+import (
+	"bytes"
+	"io"
+)
+
+// NewPtyWriter creates a writer that handles when the session has a active
+// PTY, replacing the \n with \r\n.
+func NewPtyWriter(w io.Writer) io.Writer {
```

Comment:
Do we need to export `NewPtyWriter` and `NewPtyReadWriter`?

Reply:
good question... I don't think we **need** to export, but I think it would be nicer if we do... whatever @belak decides is fine by me though.